spec/controllers/curation_concerns/file_sets_controller_spec.rb

```diff
@@ -164,8 +170,6 @@
           expect(CharacterizeJob).to receive(:perform_later).with(file_set.id, kind_of(String))
           post :update, id: file_set, file_set: { files: [file] }
           expect(response).to redirect_to main_app.curation_concerns_file_set_path(file_set)
-          skip 'pending hydra-works#89'
-          expect(file_set.reload.label).to eq 'image.png'
```

Comment:
I never was able to figure out where label was getting changed.  I think the spirit of that expect was to check that the file_set had actually been updated with the new file.  It seems that this doesn't have to be done by checking `label` but could be done by checking something after reload.

Maybe look at what update should cause to change in the file_set when it updates the files content, and check that it is actually occurring.